### DISCUSSION TITLE
多个项目放在一个工作空间里，版本信息不显示

### DISCUSSION THREAD

**Discussion by RainySecret** [pxzxj/VCSLabel#8] [General]
![image](https://user-images.githubusercontent.com/69134368/158930576-2385b89d-e789-41c3-bbfa-624dde790c80.png)
一个项目单独放一个空间里是正常显示的，但是多个项目放在一个空间里，版本信息不显示，但是以前的svnLable是可以的，大佬辛苦再看下呗
![image](https://user-images.githubusercontent.com/69134368/158930768-47e97128-e8da-49ad-8a9a-010b56c6a9e8.png)

**Comment by pxzxj** [comment]
你的.svn目录在哪的 test下面还是这个相关重要什么脚本下面

**Reply by RainySecret** [reply]
.svn在这个相关重要什么脚本下面放着，test只是一个文件夹而已，可以理解为eclipse里的workspace名字

**Reply by pxzxj** [reply]
你这个需求我觉得很多人应该都不会有，就是把很多SVN的项目放在一个非SVN项目下然后打开这个非SVN项目，所以我就不实现啦

**Reply by RainySecret** [reply]
然而大佬你在支持Git之前的版本里是可以的，也就是还叫svnLable的时候是好着的，有空的话，在后面发布版本时辛苦修复一下呗

**Reply by pxzxj** [reply]
有个最简单的办法兄弟 你只要在test下面新建一个名字叫.svn的文件夹就好了 里面不用放东西 这样插件就知道你是个svn项目了

**Reply by RainySecret** [reply]
果然可行，666，必须五星，感谢
